pkg/printers/humanreadable.go

```diff
@@ -535,6 +535,15 @@ func (h *HumanReadablePrinter) PrintTable(obj runtime.Object, options PrintOptio
 // different from lastType) including all the rows in the object. It returns the current type
 // or an error, if any.
 func printRowsForHandlerEntry(output io.Writer, handler *handlerEntry, obj runtime.Object, options PrintOptions, includeHeaders bool) error {
+	results := []reflect.Value{}
```

Comment:
this should be `var results []reflect.Value`  You don't use the value you initialize here, you simply assign to it.

Reply:
updated, really think you : )